sdks/java/extensions/sql/build.gradle

```diff
@@ -158,3 +160,26 @@ task runPojoExample(type: JavaExec) {
   classpath = sourceSets.main.runtimeClasspath
   args = ["--runner=DirectRunner"]
 }
+
+
+task integrationTest(type: Test) {
+  group = "Verification"
+  def gcpProject = project.findProperty('gcpProject') ?: 'apache-beam-testing'
+
+  systemProperty "beamTestPipelineOptions", JsonOutput.toJson([
+          "--project=${gcpProject}",
+          "--blockOnRun=false"
+  ])
+
+  include '**/*IT.class'
+  maxParallelForks 4
+  classpath = project(":beam-sdks-java-extensions-sql").sourceSets.test.runtimeClasspath
+  testClassesDirs = files(project(":beam-sdks-java-extensions-sql").sourceSets.test.output.classesDirs)
+  useJUnit { }
+}
+
+task postCommit {
```

Comment:
nit that is not your problem - I think mostly that modules should define suites and the top level should choose pre/post commit